redis-om-spring/src/main/java/com/redis/om/spring/search/stream/ReturnFieldsSearchStreamImpl.java

```diff
@@ -359,4 +360,12 @@ public <R> AggregationStream<R> groupBy(MetamodelField<T, ?>... field) {
     throw new UnsupportedOperationException("load is not supported on a ReturnFieldSearchStream");
   }
 
+  @Override public Optional<T> min(NumericField<T, ?> field) {
+    throw new UnsupportedOperationException("min is not supported on a ReturnFieldSearchStream");
```

Comment:
you can't do an orderby and then return the top record?

Reply:
I'll experiment with it and do another PR. I guess I would have to check that the `field` is already in the returned set also.